cql3/CqlParser.g4

```diff
@@ -637,11 +644,11 @@ columnList
    ;
 
 expressionList
-   : (constant | assignmentMap | assignmentSet | assignmentList) (syntaxComma (constant | assignmentMap | assignmentSet | assignmentList))*
+   : (constant | assignmentMap | assignmentSet | assignmentList | assignmentTuple) (syntaxComma (constant | assignmentMap | assignmentSet | assignmentList | assignmentTuple))*
    ;
 
 select
-   : kwSelect (distinctSpec |) selectElements fromSpec whereSpec? orderSpec? limitSpec? allowFilteringSpec?
+   : kwSelect (distinctSpec |) (jsonSpec |) selectElements fromSpec whereSpec? orderSpec? limitSpec? allowFilteringSpec?
```

Comment:
```g4
: kwSelect distinctSpec? jsonSpec? selectElements fromSpec whereSpec? orderSpec? limitSpec? allowFilteringSpec?
```

Reply:
The `( x |)` grammar are now `x?`, neat and easy to understand.